### PR TITLE
Poll heading and answers text size change and provided underline

### PR DESCRIPTION
**Pull request by techaffinity**
Poll title and its options text has been changed and provided the underline for the link "Terms and Conditions" on the registration page.

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def __init__(self, *args, **kwargs):
         self.fields['email'].label = "Choose email"
         self.fields['password1'].label = "Choose a PIN"
 
-        self.fields['tos'].label = mark_safe('I accept the <a href="%s">terms '
+        self.fields['tos'].label = mark_safe('I accept the <a href="%s" style="text-decoration:underline;">terms '
```
Comment (miltontony) [inline app/forms.py]: Don't use inlines. Please add `.form-register a{text-decoration:underline}` instead.

File: project/templates/app/mobi/poll/inclusion_tags/poll_widget.html [hunk 2]
```diff
@@ -4,8 +4,8 @@
     {% if can_vote_on_poll and not READ_ONLY_MODE %}
         <form action="{% url poll_vote object.slug %}" method="POST" id="poll-widget-form-{{ object.id }}" class="poll highlight-quaternary">
         {% csrf_token %}
-            <h3 class="article-title">{{ object.title }}</h3>
-            <div class="poll-options">
+            <h3 class="article-title poll-article-heading-text">{{ object.title }}</h3>
+            <div class="poll-options" style="font-size:1em;">
```
Comment (miltontony) [inline project/templates/app/mobi/poll/inclusion_tags/poll_widget.html]: No inlines. Please update in css.

**Comment by techaffinity** [comment]
Inline styles has been removed abd css also been updated.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**